guide/easy-install.md

```diff
@@ -0,0 +1,146 @@
+# Installing Moonfire NVR using provided scripts
+
+There are no binary packages of Moonfire NVR available yet, so it must be built
+from source. This is made easy using a few scripts that will do the job for you
+unless you have very a different operating system. The scripts are written and
+tested under ubuntu and raspbian but should not be hard to modify if necessary.
+You'll start by downloading moonfire if you have not already done so.
+
+## Downloading
+
+See the [github page](https://github.com/scottlamb/moonfire-nvr) (in case
+you're not reading this text there already). You can download the bleeding
+edge version from the command line via git:
+
+    $ git clone https://github.com/scottlamb/moonfire-nvr.git
+
+## Preparation steps for easy install
+
+There are a few things to prepare if you want a truly turnkey install, but
+they are both optional.
+
+### Dedicated media directory
+
+An optional, but strongly suggested, step is to setup a dedicated hard disk
+for recording video.
+Moonfire works best if the video samples are collected on a hard drive of
+sufficient capacity and separate from the root and main file systems. This
+is particularly important on Raspberry Pi based systems as the flash based
+main file systems have a limited lifetime and are way too small to hold
+any significant amount of video.
+If a dedicated hard drive is available, set up the mount point (in this 
+example we'll use /media/nvr/samples):
```

Comment:
could everything throughout use the singular term "sample file dir" as opposed to "samples dir" or "samples media dir"? (here, install-manual.sh, the script-functions.sh, build.sh.) just seems less confusing to be consistent; less likely people will misremember the commandline flag name and such then.